src/plugins/mf_priority.cpp

```diff
@@ -798,47 +809,29 @@ static int validate_cb (flux_plugin_t *p,
         return flux_jobtap_reject_job (p, args, "unable to unpack bank arg");
     }
 
-    // make sure user belongs to flux-accounting DB; there are two behaviors
-    // supported in this plugin:
-    //
-    // if the plugin has SOME data about users/banks and the user does not
-    // have an entry in the plugin, the job will be rejected.
-    //
-    // if the plugin has NO data about users/banks and the user does not have
-    // an entry in the plugin, the job will be held until data is received by
-    // the plugin.
-    it = users.find (userid);
-    if (it == users.end ()) {
-        // check if the map only contains DNE entries
+    // perform a lookup in the users map of the unpacked user/bank
+    a = get_association (userid, bank, users, users_def_bank);
+
+    if (a == nullptr) {
+        // the assocation could not be found in the plugin's internal map,
+        // so perform a check to see if the map has any loaded
+        // flux-accounting data before rejecting the job
         bool only_dne_data = check_map_for_dne_only ();
 
         if (users.empty () || only_dne_data) {
             add_missing_bank_info (p, h, userid);
             return 0;
         } else {
-            return flux_jobtap_reject_job (p, args,
-                                    "no bank found for user: %i", userid);
+            return flux_jobtap_reject_job (p,
+                                           args,
+                                           "cannot find user/bank or "
+                                           "user/default bank entry "
+                                           "for: %i", userid);
```

Comment:
Maybe add `for uid: %i`? Just to clarify the number that follows is a uid.

Reply:
That's a great suggestion, thanks; will fix this.